app/views/shop/products/_searchbar.haml

```diff
@@ -3,10 +3,11 @@
     .small-12.large-5.columns.flex
       %div.search-wrap
         %input#search.text{"ng-model" => "query",
+              type: 'search',
               placeholder: t(:products_search),
               "ng-debounce" => "200",
               "ofn-disable-enter" => true}
-        %a.clear{type: 'button', ng: {show: 'query', click: 'clearQuery()'}}
+        %a.clear{'focus-search' => true, type: 'button', ng: {show: 'query', click: 'clearQuery()'}}
```

Comment:
I think focus-search, a detail about the behavior, would be better at the end of the attributes list after type and click behavior.